doc/cygnus-ngsi/installation_and_administration_guide/backends_as_sth.md

```diff
@@ -0,0 +1,40 @@
+#<a name="top"></a>Backends as short-term historics
+Backends are used by Cygnus NGSI as "infinite" context data repositories. More and more data is appended to files, tables and collections as data flows from a NGSI source. Such a data flow may never end, thus, insertions may never end too, exhausting the available storing resources.
```

Comment:
It is plural, fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1392/commits/dcbac8a77cdfa1c2d71513679c6bf0ae71e812c8